core-v1/src/main/scala/data/Justified.scala

```diff
@@ -125,6 +126,25 @@ object Justified {
 
   implicit def extractValue[V]: ExtractValue[Justified[V], V] = _.value
 
+  override val typeClass: ProductTypeClass[Justified] = new ProductTypeClass[Justified] {
+
+    override def product[H, T <: HList](
+      ch: Justified[H],
+      ct: Justified[T],
+    ): Justified[H :: T] = Justified.byInference("::", ch.value :: ct.value, NonEmptyList.of(ch, ct))
+
+    override def emptyProduct: Justified[HNil] = Justified.byConst(HNil)
+
+    override def project[F, G](
+      instance: => Justified[G],
+      to: F => G,
+      from: G => F,
+    ): Justified[F] = {
+      val justified = instance
+      Justified.byInference("map", from(justified.value), NonEmptyList.of(justified))
+    }
+  }
```

Comment:
I don't think this is used anywhere. I didn't really understand how to use this. It seems like it is more for typeclass derivation for ADTs, but I don't really need to derive `Justified` instances for arbitrary types.